### ISSUE TITLE
Specify the name for container port 8181

### ISSUE THREAD

**Issue by DpoBoceka** [megaease/easeprobe#311]
[Container port](https://github.com/megaease/easeprobe/blob/v2.0.1/helm/easeprobe/templates/deployment.yaml#L43-L44) does not have an associated name to it. It's used for prometheus scrapping jobs and traditionally is **metrics**

**Comment by samanhappy** [comment]
Thanks for reporting, we have automatically scraped metrics for EaseProbe by [adding extraScrapeConfigs](https://github.com/megaease/easeprobe/blob/main/helm/easeprobe/values.yaml#LL43-L49) , and you can see them in Prometheus like 
![WX20230315-114406@2x](https://user-images.githubusercontent.com/2755122/225201644-85a36b9e-aa35-49f1-9c65-a6280e9afa6d.png)
Does it meet your needs?

**Comment by DpoBoceka** [comment]
Not quite, my prometheus instance is deployed to another cluster and as a scrapper I use opentelemetry-collector. Either way, I'd prefer setting a scrapping job in collector's context via `monitoring.coreos.com/v1` `ServiceMonitor` CRD, but it looks like it works with port _names_ only
